src/utils/constants.ts

```diff
@@ -60,3 +60,14 @@ export const SORT_FIELDS: ReadonlyArray<string> = [
 	'username:asc',
 	'username:desc',
 ];
+
+export enum NETWORK {
```

Comment:
why are we using `enum` here?

Reply:
use of constants doesn't make sense here since we will only support few networks and some places we have `main`, `mainnet` like different format, so I thought have this way it helps.